### DISCUSSION TITLE
Can I use my discord account through Revolt?

### DISCUSSION THREAD

**Discussion by rado84-github** [stoatchat/discussions#551] [Q&A] [answered]
IDK if I asked it properly, so let me explain. I have a discord account with over 25 servers in it (they're not mine, I'm just their member). But lately discord became too glitchy, the developers too lazy and on top of that - too greedy. So I'm looking for a way to login to my discord account and have access to all of the servers I'm a member of, just without actually using the discord app.
So, is that possible with revolt?

**Comment by Zomatree** [comment, marked as answer]
Thats not possible with Revolt, they are completely separate.

**Reply by rado84-github** [reply]
Pity. Then Revolt is of no use to me. It will be impossible to make the owners of these servers to move to Revolt, no matter how good its description sounds.
